src/app/services/helpers/dateNowFormat.ts

```diff
@@ -0,0 +1,9 @@
+export const dateNowFormat = () => {
```

Comment:
better to format any Date passed as an argument and later call it with `new Date()`